__tests__/infrastructure/AccountService.spec.js

```diff
@@ -1,26 +1,175 @@
+
+		it('return account', async () => {
+			// Arrange:
+			const account = TestHelper.generateAccount(1)[0];
+			const mockAccountInfo = TestHelper.mockAccountInfo(account);
+
+			const mockChainInfo = {
+				latestFinalizedBlock: {
+					height: 187732,
+					hash: '6E95A894530C3E55EA8801A3715C6E247BB4B76F41721D41E21110E211E0231D',
+					finalizationPoint: 39,
+					finalizationEpoch: 300
+				}
+			};
+
+			const mockAccountAlias = [{
+				address: account.address.plain(),
+				names: [{
+					name: 'alias',
+				}]
+			}];
+
+			getAccount.returns(Promise.resolve(mockAccountInfo));
+
+			getChainInfo.returns(Promise.resolve(mockChainInfo));
+
+			getAccountsNames.returns(Promise.resolve(mockAccountAlias));
+
+			// Act:
+			const accountInfo = await AccountService.getAccountInfo(account.address.plain());
+
+			// Assert:
+			expect(accountInfo.address).toEqual(account.address.plain());
+			expect(accountInfo.votingList[0].epochInfo.epochStatus).toEqual('Current');
+			expect(accountInfo.votingList[1].epochInfo.epochStatus).toEqual('Future');
+			expect(accountInfo.votingList[2].epochInfo.epochStatus).toEqual('Expired');
+			expect(accountInfo.accountAliasNames).toEqual(['alias']);
+		});
+	});
+
+	describe('getAccountList should', () => {
+		const pageInfo = {
+			pageNumber: 1,
+			pageSize: 10
+		};
+		let searchAccounts = {};
+		let getStorageInfo = {};
+		let getAccountsNames = [];
+
+		beforeEach(() => {
+			searchAccounts = stub(AccountService, 'searchAccounts');
+			getAccountsNames = stub(NamespaceService, 'getAccountsNames');
+			getStorageInfo = stub(NodeService, 'getStorageInfo');
+
+			getStorageInfo.returns(Promise.resolve({
+				numAccounts: 100
+			}));
+		});
+
+		afterEach(restore);
+
+		/**
+		 * setup mock account list.
+		 * @param {array} mosaics mosaic list.
+		 * @param {boolean} isAccountAlias true to set account alias.
+		 */
+		const setupMockAccounts = (mosaics = [], isAccountAlias = false) => {
+			const accounts = TestHelper.generateAccount(10);
+
+			const mockSearchAccounts = {
+				...pageInfo,
+				data: accounts.map(account => {
+					return {
+						...TestHelper.mockAccountInfo(account),
+						mosaics
+					};
+				})
+			};
+
+			const mockAccountsAlias = accounts.map((account, index) => {
+				const name = isAccountAlias ? {
+					name: `alias_${index + 1}`
+				}  : {};
+
+				return {
+					address: account.address.plain(),
+					names: [name]
+				};
+			});
+
+			searchAccounts.returns(Promise.resolve(mockSearchAccounts));
+
+			getAccountsNames.returns(Promise.resolve(mockAccountsAlias));
+		};
+
+		it('return accounts', async () => {
+			// Arrange:
+			setupMockAccounts();
+
+			// Act:
+			const accountList = await AccountService.getAccountList(pageInfo, {});
+
+			// Assert:
+			expect(accountList.totalRecords).toEqual(100);
+			expect(accountList.pageNumber).toEqual(pageInfo.pageNumber);
+			expect(accountList.pageSize).toEqual(pageInfo.pageSize);
+			accountList.data.forEach(account => {
```

Comment:
in this test and following ones, you should at least check number of returned elements...